src/routes/pat/pat.ts

```diff
@@ -16,10 +16,6 @@ export const createPATHandler: RequestHandler<
   {},
   { metadata: object; expiresAt: Date }
 > = async (req, res) => {
-  if (!req.auth) {
-    return sendError(res, 'unauthenticated-user');
-  }
-
```

Comment:
Why did we remove this check? I think it's not the same as 'user-not-found'

Reply:
I added the middleware here; which should cover this + the elevation if needed:

https://github.com/nhost/hasura-auth/pull/462/files#diff-0cab04d0c3bdd1b5644679487264bae0931d9a2113af26dc7236bf3977da7137R18